exonum/benches/block.rs

```diff
@@ -122,12 +124,13 @@ mod tests {
                 self.verify_signature(self.from())
             }
 
-            fn execute(&self, view: &mut Fork) {
-                let mut index = ProofMapIndex::new("balances_txs", view);
+            fn execute(&self, fork: &mut Fork) -> TransactionResult {
+                let mut index = ProofMapIndex::new("balances_txs", fork);
                 let from_balance = index.get(self.from()).unwrap_or(0u64);
                 let to_balance = index.get(self.to()).unwrap_or(0u64);
                 index.put(self.from(), from_balance - 1);
                 index.put(self.to(), to_balance + 1);
+                Ok(TransactionValue::Success)
```

Comment:
In my opinion `TransactionValue` looks somewhat incomprehensible for service developers.